addons/account/views/report_invoice.xml

```diff
@@ -123,7 +123,7 @@
 
                     <div class="clearfix">
                         <div id="total" class="row">
-                            <div t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'} ml-auto">
+                            <div t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'} ml-auto mr-0">
```

Comment:
The only change between 14.0 and 15.0 in that part of the code was https://github.com/odoo/odoo/commit/d9a3b938fe1e3f3311574641b9eeed3c47a19e58
Since this specific line didn't change between the versions, I feel like the change of class should be somewhere in `account.document_tax_totals` instead. At least, it is missing in the Purchase Order.

Reply:
Hello @william-andre , thanks for your review :slightly_smiling_face: 

After checking a Purchase Order, it turns out that the layout is good. Indeed, used classes are different in this case: 
https://github.com/odoo/odoo/blob/1f652aaa97b154b2b7c3136cb1cf7c7588501540/addons/purchase/report/purchase_order_templates.xml#L111-L112

I'm not a CSS Expert but I would say that we should finally remove the notions of margins (`ml-auto` and my addition `mr-0`) and use the `justify-content-end` class on the parent: which would make it easier to manage the alignment.

What do you think about it ?